src/components/DatePicker.tsx

```diff
@@ -0,0 +1,68 @@
+import './styles/date_picker.less'
+
+import React from 'react'
+
+import Input from './Input'
+import Pikaday from 'pikaday'
+import dayjs from 'dayjs'
+import CalendarIcon from './icons/CalendarIcon.svg'
+import classNames from 'classnames'
+
+type Props = {
+    /* id assigned to the picker */
+    id?: string
+    /* function to handle when the date changes */
+    onDateChanged?: (selectedDate: Date) => void
+    /* placeholder text to display before a date picked*/
+    placeholder?: string
+    /* minimum date permitted */
+    min?: string
+    /* maximum date permitted */
+    max?: string
+    defaultValue?: string
+    /* Additional css class that is applied to DatePicker */
+    className?: string
+}
+
+class DatePicker extends React.Component<Props> {
+    componentDidMount(): void {
+        const { onDateChanged, min, max } = this.props
+
+        new Pikaday({
+            theme: 'pika-twist-composer-input',
```

Comment:
reactist shouldn't contain explicit twist references

Reply:
As this component has been removed for now, this is now resolved.